bindings/Sofa/package/__init__.py

```diff
@@ -76,7 +76,19 @@
     sofa_bin_path = os.path.join(sofa_root, "bin")
     sofapython3_bin_path = os.path.join(sofapython3_root, "bin")
 
-    compilation_modes = ["Release", "RelWithDebInfo", "Debug", "MinSizeRel"]
+    # A user using a build configuration could have a multiple-configuration type build
+    # which is typical on Windows and MSVC; and MacOS with XCode
+    # Previously the setup was arbitrarily choosing the first configuration it could find.
+    # Now, if the user set the env.var SOFA_BUILD_CONFIGURATION, he can choose a preferred configuration.
```

Comment:
I would not mention the previous behavior in a comment. Just focus on the current behavior